### PR TITLE
ocf_desktop: purge ayatana-indicator-application

### PR DESCRIPTION
**Pull request by kpengboy**

### CODE REVIEW

**Comment by kpengboy** [comment]
Do you all think I should explicitly install remmina without any dependencies or is what I have fine

**Comment by dkess** [comment]
Can you give more context on what this is? What do we use it for?

**Comment by kpengboy** [comment]
Installing remmina pulls in ayatana-indicator-application as an indirect recommended dependency.

Without ayatana-indicator-application:

Without ayatana-indicator-application

**Comment by kpengboy** [comment]
With ayatana-indicator-application:

With ayatana-indicator-application

File: modules/ocf_desktop/manifests/packages.pp [hunk 1]
```diff
@@ -53,12 +53,15 @@
 
   # Remove some packages
   package {
+    # causes the fcitx menu to vanish
```
Comment (jvperrin) [inline modules/ocf_desktop/manifests/packages.pp]: Maybe say what the fcitx menu does for those who don't know it is for switching keyboard layouts?

**Pull request merged**